coeus-impl/src/main/java/org/kuali/coeus/propdev/impl/budget/core/ProposalBudgetForm.java

```diff
@@ -202,4 +203,12 @@ public void setDataValidationItems(List<DataValidationItem> dataValidationItems)
 		this.dataValidationItems = dataValidationItems;
 	}
 
+	public boolean isViewOnly() {
+		return viewOnly;
+	}
+
+	public void setViewOnly(boolean viewOnly) {
+		this.viewOnly = viewOnly;
+	}
```

Comment:
Yeah, I tried that initially but I see that budget goes readOnly when it is set as complete.
So thought better not rely on that.